src/prost/src/lib.rs

```diff
@@ -551,4 +562,11 @@ mod tests {
         };
         assert!(!new_data_type.is_nullable);
     }
+
+    #[test]
+    fn test_size() {
+        use static_assertions::const_assert_eq;
+
+        const_assert_eq!(std::mem::size_of::<NodeBody>(), 16);
```

Comment:
May add some comments to instruct future developers to box the newly introduced variant if this assertion fails.

Also, can use `static_assertions` from the standard library and there's no need to put it inside a `#[test]` function.

Reply:
Where is the static_assertions in the standard library ?

Reply:
Oh, there's not. I confused it with `assert_matches`. 😄